client/src/login/LoginForm.js

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { Field, reduxForm } from 'redux-form';
+
+export const LoginForm = reduxForm({
```

Comment:
Using the reduxForm HOC this way is pretty atypical. However I was also having some trouble with it yesterday so I think something must have changed since the previous version. I'm going to look in to it and see if we can get a more standard syntax working but for now if this works it's fine :)